java/configuration/src/main/java/sleeper/configuration/properties/instance/DefaultProperty.java

```diff
@@ -257,6 +257,13 @@ public interface DefaultProperty {
                     "is large.")
             .defaultValue("async")
             .propertyGroup(InstancePropertyGroup.DEFAULT).build();
+    UserDefinedInstanceProperty DEFAULT_COMPACTION_JOB_ID_ASSIGNMENT_COMMIT_ASYNC = Index.propertyBuilder("sleeper.default.compaction.job.id.assignment.commit.async")
+            .description("If true, compaction job ID assignment commit requests will be sent to the state store committer lambda " +
+                    "to be performed asynchronously. If false, compaction job ID assignments will be committed " +
+                    "synchronously in the compaction job creation lambda.")
+            .defaultValue("false")
```

Comment:
It looks like this should default to true? It also looks like the ingest aync commit property is set to false incorrectly.

Since the default state store is the transaction log implementation, I think we should send every state store update to the lambda by default.

Reply:
This might break a lot of tests, maybe we should make that a new issue?